### PR TITLE
Add Q&A app

### LINKED ISSUE #16: Add Question Answering Web App sample application
Add https://github.com/IBM/MAX-Question-Answering-Web-App to https://github.com/CODAIT/max-central-repo/blob/master/max-model-status.md

### PR DESCRIPTION
**Pull request by SSaishruthi**
closes #16

### CODE REVIEW

File: max-model-status.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Current status for the https://developer.ibm.com/exchanges/models/ ecosystem.
 
 | Name | Status | Latest Release |
 | ---- | ---- | ---- | 
+| [Question Answering Web App](https://github.com/IBM/MAX-Question-Answering-Web-App) | [![Website](https://img.shields.io/website/http/max-question-answering-web-app.max.us-south.containers.appdomain.cloud/app.svg)](http://max-question-answering-web-app.max.us-south.containers.appdomain.cloud)| [![GitHub release](https://img.shields.io/github/v/release/IBM/MAX-Question-Answering-Web-App.svg)](https://github.com/IBM/MAX-Question-Answering-Web-App/releases/latest) |
```
Comment (SSaishruthi) [inline max-model-status.md]: @ptitzler How do we get the website `svg` link?

Comment (xuhdev) [reply max-model-status.md]: No build status?

Comment (ptitzler) [reply max-model-status.md]: The badge link should be `https://img.shields.io/website/http/max-question-answering-web-app.max.us-south.containers.appdomain.cloud.svg`. However the q&a app itself has an issue that needs to be fixed to make this work. It's my understanding that shields.io runs the equivalent of `curl -I http://max-question-answering-web-app.max.us-south.containers.appdomain.cloud` which currently raises HTTP error 500. If you try this with our other deployed apps you should see that they all return a positive status code.

Comment (ptitzler) [reply max-model-status.md]: Just open two issues in https://github.com/IBM/MAX-Question-Answering-Web-App

Comment (ptitzler) [reply max-model-status.md]: https://github.com/IBM/MAX-Question-Answering-Web-App/issues/14
https://github.com/IBM/MAX-Question-Answering-Web-App/issues/15

**Review by ptitzler — APPROVED** [review]
LGTM. The build and availability badges will be added once the issues mentioned in this PR are resolved.

**Pull request merged**